### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by rov9**

### CODE REVIEW

**Comment by rov9** [comment]
Kindly review.Live url:https://loving-clarke-16fd02.netlify.app/

File: style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    border-radius: 2px;
+    text-transform: capitalize;
+    border: none;
+    cursor: pointer;
+    display: block;
+    width: 300px;
+    height: 40px;
+    text-align: center;
+    line-height: 40px;
+    background: #383838;
```
Comment (krishanu2911) [inline style.css:44]: declare the css variable

File: style.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+body{
+    
+    display: flex;  
+    justify-content: center;
+    align-items: center;
```
Comment (krishanu2911) [inline style.css:5]: you can move this properties to utils

**Review by krishanu2911 — APPROVED** [review]
Added a few suggestions Do make these changes

**Comment by rov9** [comment]
Authorization - Sign Up

**Pull request merged**